test/test_workspaces.py

```diff
@@ -41,3 +41,18 @@ def test_order_paths(self):
         self.assertEqual(['foo', 'bar'], order_paths(['bar', 'foo'], ['foo']))
         self.assertEqual(['baz', 'foo', 'bar'], order_paths(['bar', 'foo', 'baz'], ['baz', 'foo']))
         self.assertEqual(['foo' + os.sep + 'bim', 'bar'], order_paths(['bar', 'foo' + os.sep + 'bim'], ['foo']))
+
+    @unittest.skipUnless(hasattr(os, 'symlink'), 'requires symlink availability')
+    def test_order_paths_with_symlink(self):
+        root_dir = tempfile.mkdtemp()
+        try:
+            foo = os.path.join(root_dir, 'foo')
+            foo_inc = os.path.join(root_dir, os.path.join(foo, 'include'))
```

Comment:
This change is incorrect. Please use my previous suggestion: https://github.com/ros-infrastructure/catkin_pkg/pull/260#discussion_r307357582

Reply:
Obviously - corrected.